### PR TITLE
add posit (tapered float) core type implementation

### PR DESCRIPTION
**Pull request by 6293**
https://github.com/internet2-org/rust-aluvm/issues/5

about Posit: https://posithub.org/docs/Posits4.pdf

### CODE REVIEW

File: num/src/posit.rs [hunk 1]
```diff
@@ -0,0 +1,160 @@
+// Rust language amplification library providing multiple generic trait
+// implementations, type wrappers, derive macros and other language enhancements
+//
+// Written in 2014 by
+//     Andrew Poelstra <apoelstra@wpsoftware.net>
+// Refactored & fixed in 2021 by
+//     Dr. Maxim Orlovsky <orlovsky@pandoracore.com>
```
Comment (dr-orlovsky) [inline num/src/posit.rs:7]: Please put your name here instead of these two

**Review by dr-orlovsky — APPROVED** [review]
ACK 7f2523e652599e8c5b6c4a14876d0995948dd50a

File: num/src/posit.rs (hunk 1 above)
Comment (6293) [reply num/src/posit.rs]: done

**Review by dr-orlovsky — APPROVED** [review]
ACK c8d55ffc38f91e1d6cd8d26fcd8825d32915a2b1

**Pull request merged**